Seminar-IS/Seminar02/Pract 02/task01.cpp

```diff
@@ -0,0 +1,45 @@
+#include <iostream>
+#include <fstream>
+
+using namespace std;
+
+void saveNumbersToFile() {
+    int* nums = new int[3];
+    for (size_t i = 0; i < 3; i++) {
+        cin >> nums[i];
+    }
+
+    int sum = 0;
+    int product = 1;
+    for (size_t i = 0; i < 3; i++) {
+        sum += nums[i];
+        product *= nums[i];
+    }
+
+    ofstream outFile("result.txt");
```

Comment:
Use const char FILE_NAME[] for example to store the name of the file and use it instead of writing "result.txt" every time.